core/site_scons/models/T3W1/trezor_t3w1_revA.py

```diff
@@ -84,6 +84,7 @@ def configure(
         features_available.append("ble")
         defines += [("USE_BLE", "1")]
         sources += ["embed/io/nrf/stm32u5/nrf.c"]
+        sources += ["embed/io/nrf/stm32u5/nrf_test.c"]
```

Comment:
this probably should be done for A0 too, but i think i will drop this revision soon anyway so we can keep it like this, just noting